### PR TITLE
ADD Kubernetes 1.17.3 and Docker 19.03.4, remove deprecated Hyperkube

### PR DESCRIPTION
**Pull request by mzaneri**

### CODE REVIEW

File: packages/kubernetes/1.16.4/Manifest [hunk 1]
```diff
@@ -1,4 +1,8 @@
-image kubernetes k8s.gcr.io/hyperkube:v1.16.4
+image kube-apiserver k8s.gcr.io/kube-apiserver:v1.16.4
+image kube-controller-manager k8s.gcr.io/kube-controller-manager:v1.16.4
+image kubes-cheduler k8s.gcr.io/kube-scheduler:v1.16.4
+image kube-proxy k8s.gcr4io/kube-proxy:v1.16.4
```
Comment (areed) [inline packages/kubernetes/1.16.4/Manifest]: There's a `4` in there

File: scripts/kustomize/kubeadm/init-orig/kubeadm-cluster-config-v1beta2.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ kubernetesVersion: v$KUBERNETES_VERSION
 certificatesDir: /etc/kubernetes/pki
 clusterName: kubernetes
 controllerManager: {}
-useHyperKubeImage: true
+useHyperKubeImage: false
```
Comment (areed) [inline scripts/kustomize/kubeadm/init-orig/kubeadm-cluster-config-v1beta2.yml]: Is this going to work for 1.15 airgap installs that still have hyperkube in the airgap pacakge?

Comment (mzaneri) [reply scripts/kustomize/kubeadm/init-orig/kubeadm-cluster-config-v1beta2.yml]: I think the best solution for me here is to remove hyperkube for the 1.15 installs as well

File: packages/kubernetes/1.16.4/Manifest [hunk 3]
```diff
@@ -1,4 +1,7 @@
-image kubernetes k8s.gcr.io/hyperkube:v1.16.4
+image kube-apiserver k8s.gcr.io/kube-apiserver:v1.16.4
+image kube-controller-manager k8s.gcr.io/kube-controller-manager:v1.16.4
+image kubes-cheduler k8s.gcr.io/kube-scheduler:v1.16.4
+image kube-proxy k8s.gcrio/kube-proxy:v1.16.4
```
Comment (areed) [inline packages/kubernetes/1.16.4/Manifest]: gcr.io

File: scripts/kustomize/kubeadm/init-orig/kubeadm-cluster-config-v1beta2.yml (hunk 2 above)
Comment (mzaneri) [reply scripts/kustomize/kubeadm/init-orig/kubeadm-cluster-config-v1beta2.yml]: changed

File: packages/kubernetes/1.16.4/Manifest (hunk 3 above)
Comment (mzaneri) [reply packages/kubernetes/1.16.4/Manifest]: changes

**Pull request merged**